rats-devtools/src/python/rats/docs/_commands.py

```diff
@@ -20,34 +24,70 @@ class RatsCiCommands(apps.AnnotatedContainer):
     def __init__(self, app: apps.Container) -> None:
         self._app = app
 
+    @apps.group(command_tree.CommandTreeServices.GROUPS.subcommands("rats-devtools docs"))
```

Comment:
with my plugin classes, i think i've separated the command from where it gets assigned to a parent command. is that possible here? can we avoid mentioning the "rats-devtools" and "docs" commands?

Reply:
That's 100% possible! I just chose the `rats-devtools docs` as the "subcommands" group because it makes it easy to see where the command fits in with everything else when looking at a single function

You could replace it with an arbitrary value, as long as it's a group ServiceId that subcommands are registered to

I tend to lean towards the parent command getting to decide whether to expose a subcommand group ServiceId (open for extension) or explicitly list the subcommands (closed for extension). What about you?

Reply:
let me tinker in my branch tonight and form some opinions :) i'm going to convert some of my commands to different styles like you did here and see how it compares